test/Rules/INCOME_TAX.test.js

```diff
@@ -0,0 +1,136 @@
+const expect = require('chai').expect;
+const EventEmitter = require('events');
+const sinon = require('sinon');
+const mockUIFactory = require('../mocks/UI');
+
+const { GameState } = require('../../entities/GameState');
+const { createPlayerFactory } = require('../testutils');
+const config = require('../../config/monopolyConfiguration');
+const PlayerActions = require('../../entities/PlayerActions');
+
+describe('Rules -> INCOME_TAX', () => {
+  let gameState;
+  let userInterface;
+  let eventBus;
+  const RULES = require('../../entities/Rules');
+
+  beforeEach(() => {
+    gameState = new GameState();
+    eventBus = new EventEmitter();
+    userInterface = mockUIFactory();
+    let createPlayer = createPlayerFactory();
+    gameState.players = [createPlayer({ name: 'player1' })];
+    gameState.config = config;
+  });
+
+  afterEach(() => {
+    // Restore the default sandbox here
+    sinon.restore();
+  });
+
+  describe('incomeTax', () => {
+    const inputEvent = 'INCOME_TAX';
+
+    let incomeTaxSpy;
+
+    beforeEach(() => {
+      let { emit: notify } = eventBus;
+      notify = notify.bind(eventBus);
+
+      RULES[inputEvent].forEach((handler) =>
+        eventBus.on(
+          inputEvent,
+          handler.bind(null, { notify, UI: userInterface }, gameState)
+        )
+      );
+
+      incomeTaxSpy = sinon.spy();
+      eventBus.on(inputEvent, incomeTaxSpy);
+    });
+
+    it('should make a call to the UI#incomeTaxPayment', () => {
+      const uiSpy = sinon.spy();
+      userInterface.incomeTaxPayment = uiSpy;
+      const promptStub = sinon.stub(PlayerActions, 'prompt');
+      promptStub.onCall(0).returns('FIXED');
```

Comment:
I saw these values (i.e., `'FIXED'` and `'VARIABLE'` hardcoded in the code; is there any way that we can make these configuration settings ? Cool with it as a backlog item; I don't think its as easy as throwing it in now

Reply:
I'll add an enhancement to the backlog

Reply:
#45